### DISCUSSION TITLE
was this page helpful feedback widget - custom event tracking

### DISCUSSION THREAD

**Discussion by coliff** [plausible/analytics#1913] [Anything Else]
We've got a 'Was this page helpful?' widget on our site.
![image](https://user-images.githubusercontent.com/1212885/169294351-9b9cb6c5-b5a2-4bdf-a230-bdc25e64d37a.png)

We'd like to use custom event tracking to know when users click om the thumbs up/down buttons and what page they clicked on them. Does someone have a 'recipe' for tracking that? I've seen the [code example for tracking link clicks](https://plausible.io/docs/custom-event-goals#code-example-for-tracking-link-clicks) but these are actually `<buttons>` and not links `<a>`.

**Comment by metmarkosaric** [comment]
hi @coliff! buttons would work the exact same was as link clicks in that example. you just need to add the attribute tag to the buttons and add that javascript code to your page

**Reply by metmarkosaric** [reply]
quick correction: you should actually follow the form submit example instead. the link clicks one will only work with link `<a>` elements. the form submit example is for button elements.
